src/engine/Engine.cpp

```diff
@@ -1053,7 +1053,9 @@ void Engine::initializeNetworkLevelReasoning()
     _networkLevelReasoner = _preprocessedQuery.getNetworkLevelReasoner();
 
     if ( _networkLevelReasoner )
-        _networkLevelReasoner->setTableau( _tableau );
```

Comment:
Please undo this change

Reply:
OK. And commit + push after that?

Reply:
Nope, still here... You can ad should look at the "Files changes" tab to see the actual changes

Reply:
I think I didn't understand - you want me to remove the **whole** lines (if/else) or just the topology dump? I thought you want to revert it back and remove only the dump